pages/content/posts/london-meetup/index.md

```diff
@@ -0,0 +1,34 @@
+---
+author: "Florian Scholz"
+title: "Open Web Docs meetup with Mozilla’s MDN Team in London"
+date: "2023-02-08"
+tags: ["announcements"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "On January 18-20 all of the Open Web Docs Technical Writing team and members of the OWD Governing Committee were invited to London to meet with Mozilla’s MDN Team."
+---
+
+On January 18-20 all of the Open Web Docs Technical Writing team and members of the OWD Governing Committee were invited to London to meet with Mozilla’s MDN Team. Mozilla kindly sponsored our travel and accommodation and Google hosted all of us in their London office. We are very grateful for this opportunity. It was the first time we met as a group in person given Open Web Docs was founded during the global covid pandemic. Some of us met for the first time in real life.
+
+For Open Web Docs, the most important project we contribute to is MDN Web Docs. We work closely with Mozilla, our main sponsors Google, Microsoft, Igalia, JetBrains, Canva, our 150+ individual funders, and the larger web community to maintain and innovate on the best documentation site for web developers and designers world wide.
+
+At the end of last year, we started to work more closely with Mozilla than ever before to help with community coordination and to keep the MDN contribution process running efficiently. To that end, Mozilla and OWD also created a [shared and open roadmap](https://github.com/orgs/mdn/projects/26/) for ongoing and planned work. The Open Web Docs team is the largest group of people that contributes to MDN projects (mainly content and BCD) with the highest number of contributions and we are happy to collaborate and give direct input on MDN’s information architecture, compatibility data, content structures, code examples, and other changes coming up across the whole site.
```

Comment:
"Largest group of people" or "largest team" - is it the intention to highlight the size of the team rather than the size of the contributions? It doesn't seem terribly relevant (and depending on how we count, not necessarily true).

Reply:
I'm omitting the sentence. Within the context of this paragraph it is pretty clear we are a big deal and no need to be comparative. 

```suggestion
In Q4 of 2022, Open Web Docs started working very closely with Mozilla on community coordination and keeping the MDN contribution process running efficiently and effectively. To that end, Mozilla and OWD created a [shared and open roadmap](https://github.com/orgs/mdn/projects/26/) for ongoing and planned work. The Open Web Docs team actively collaborates and gives direct input on MDN’s information architecture, compatibility data, content structures, code examples, and other changes coming impacting all sections of the MDN Web Docs site.
```